README.md

```diff
@@ -37,10 +37,15 @@ First of all you must clone the repository:
 $ git clone https://github.com/staticafi/symbiotic
 ```
 
+Next you need to update submodules:
+```
+$ cd symbiotic
+$ git submodule update --init
```

Comment:
What is the reason for such change? AFAIK it should not be necessary.

The submodules are initialised both in `build.sh` and `system-build.sh`: https://github.com/staticafi/symbiotic/blob/0534e840b4c778b343831ee69bcc7656fc2c3d28/system-build.sh#L318 https://github.com/staticafi/symbiotic/blob/0534e840b4c778b343831ee69bcc7656fc2c3d28/build.sh#L544

Reply:
I see. I'll revert this change. (Note: I think, it is unusual to build the project to obtain all of its sources).